packages/device-utils/tests/modelUtils.test.ts

```diff
@@ -0,0 +1,35 @@
+import { DeviceModelInternal } from '@trezor/connect';
+
+import { pickByDeviceModel } from '../src';
+
+describe('pickByDeviceModel', () => {
+    const OPTIONS = {
+        default: 'default',
+        [DeviceModelInternal.T1B1]: null,
+        [DeviceModelInternal.T3T1]: 'T3T1',
+    };
+
+    const fixtures = [
+        {
+            description: 'should return correct value for a given device',
+            deviceModelInternal: DeviceModelInternal.T3T1,
+            expectedResult: 'T3T1',
+        },
+        {
+            description: 'should return null when the value for the picked device is null',
+            deviceModelInternal: DeviceModelInternal.T1B1,
+            expectedResult: null,
+        },
+        {
+            description: 'should return default when deviceModelInternal is undefined',
+            deviceModelInternal: undefined,
+            expectedResult: 'default',
+        },
```

Comment:
one case is not covered

```       
 {
            description: 'should return default when deviceModelInternal is not in options',
            deviceModelInternal: DeviceModelInternal.T2B1,
            expectedResult: 'default',
        },

        ```

Reply:
b0f77a6c2f5236c6e1982285d01d9598db784427